app/renderer/components/messageBox.js

```diff
@@ -0,0 +1,165 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../../../js/components/immutableComponent')
+const Dialog = require('../../../js/components/dialog')
+const Button = require('../../../js/components/button')
+const SwitchControl = require('../../../js/components/switchControl')
+const appActions = require('../../../js/actions/appActions')
+const KeyCodes = require('../../common/constants/keyCodes')
+const config = require('../../../js/constants/config')
+const {makeImmutable} = require('../../common/state/immutableUtil')
+const {StyleSheet, css} = require('aphrodite')
+const commonStyles = require('./styles/commonStyles')
+
+class MessageBox extends ImmutableComponent {
+  constructor () {
+    super()
+    this.onClick = this.onClick.bind(this)
```

Comment:
Good call on this- I copied from another modal and it was unnecessary. It may have originally been put there to prevent the modal from closing on click... but that doesn't happen here 👍